Source/Server/.dolittle/bounded-context.json

```diff
@@ -3,11 +3,15 @@
     "boundedContext": "f39b1f61-d360-4675-b859-53c05c87c0e6",
     "boundedContextName": "[UNSET]",
     "resources": {
+        "eventStore": {
+            "production": "MongoDB",
+            "development": "MongoDB"
+        },
         "projections": {
             "production": "MongoDB",
             "development": "MongoDB"
         },
-        "eventStore": {
+        "embeddings": {
```

Comment:
Is this file used by the new stuff (projections, embeddings) still? I know that it's required for the Runtime to boot, but I thought that it was only used by some old stuff.

Reply:
Yes it is used by the resource system for hooking up services with the correct implementation types (which we don't really use since we just have MongoDB...) But it need to stay for the time being